governance/votes/Executive vote - January 24, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Template - [Executive Vote] Temporarily prevent FLAP auctions and MKR Burn - January 24, 2022
+summary: Increase the System Surplus Buffer to 250 million DAI to prevent FLAP auctions until the auction/burn mechanism can be improved.
+date: 2022-01-24T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Temporarily prevent FLAP auctions and MKR Burn - January 24, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an out-of-schedule executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+This is an out-of-schedule executive proposal. It has been deployed and added to the voting portal in response to the large wave of FLAP auctions triggered by liquidations at the end of last week and the following [urgent signal request](https://forum.makerdao.com/t/urgent-signal-request-stop-mkr-burning/12806). 
```

Comment:
Is it worth linking to the relevant MIPs here for clarity around the governance process?

Reply:
Yep, good call.